js/status.js

```diff
@@ -27,8 +27,11 @@ function StatusMatrix() {
         docById('statusmatrix').style.border = 2;
 
         // FIXME: make this number based on canvas size.
+	
+        var canvas = docById('myCanvas')
```

Comment:
And a more verbose commit message would be helpful.

Fixes #361 by using canvas width in calculation of cell scaling factor/